packages/eslint-plugin/src/rules/no-unsafe-assignment.ts

```diff
@@ -298,8 +304,8 @@ export default createRule({
         node: reportingNode,
         messageId: 'unsafeAssignment',
         data: {
-          sender: checker.typeToString(sender),
-          receiver: checker.typeToString(receiver),
+          sender: '`' + checker.typeToString(sender) + '`',
+          receiver: '`' + checker.typeToString(receiver) + '`',
```

Comment:
[Question] Is there a reason this `data` object isn't using `creatDataFromSenderType`? I don't remember 🤔

Reply:
@tobySolutions you marked this as resolved but I don't know what the answer is 😂 what is the resolution here? What's up?